<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="plot_picture_box">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>900</width>
      <height>900</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="action_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Manjari</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Draw!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="left_bound_text_box">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="right_bound_text_box">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="y_axis_picture_box">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>91</width>
      <height>901</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="x_axis_picture_box">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>940</y>
      <width>900</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="left_bound_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Manjari</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Left:</string>
    </property>
   </widget>
   <widget class="QLabel" name="right_bound_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Manjari</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Right:</string>
    </property>
   </widget>
   <widget class="QLabel" name="enter_bounds_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Manjari</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Segment bounds</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="function_text_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="function_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Manjari</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Function</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Manjari</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 15px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
